doc/api/stream.md

```diff
@@ -774,7 +774,7 @@ end
 *Note*: In general, the `readable.pipe()` and `'data'` event mechanisms are
 easier to understand than the `'readable'` event.
 However, handling `'readable'` might result in increased throughput.
-
+in the Meanwhile, new data will not trigger `'readable'` event everytime.
```

Comment:
“In” should be capitalized. ‘readable’ is emitted if ‘read()’ returned null the last time it was called (from memory, please verify).